src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -94,6 +94,30 @@ export class ReasonForAppointmentPage extends React.Component {
             onBack={this.goBack}
             pageChangeInProgress={pageChangeInProgress}
           />
+          <AlertBox
+            headline="If you are experiencing a medical emergency:"
+            className="vads-u-margin-y--3"
+            content={
+              <ul>
+                <li>
+                  Call <a href="tel:911">911</a>,{' '}
+                  <span className="vads-u-font-weight--bold">or</span>
+                </li>
+                <li>
+                  Call the Veterans Crisis hotline at{' '}
+                  <a href="tel:800-273-8255">800-273-8255</a> and press 1,{' '}
```

Comment:
I think we typically remove dashes in phone numbers in `href`. Not positive it matters, though.